css/style.css

```diff
@@ -0,0 +1,302 @@
+body {
+  font-family: "Montserrat", sans-serif;
+  
+}
+
+h1 {
+  font-size: 32px;
+}
+
+h2 {
+  font-size: 28px;
+
+
+}
+
+h3 {
+  font-size: 25px;
+}
+
+h5 {
+  font-size: 16px;
+}
+
+p {
+  font-size: 16px;
+  font-weight: 100;
+}
```

Comment:
Jeśli reset.css nie wystarcza Ci to proponuję utworzyć też `global.css` i też go mieć oddzielnie, w ten sposób łatwiej go wykorzystać w wielu projektach :)